ridalot2.0/ridalot2.0/Pages/Feed.razor

```diff
@@ -34,25 +35,46 @@ Weight: @post.Weight
                 </td>
                 <td>@post.Pay</td>
                 <td>@post.Address</td>
-                <td> <button class="btn btn-primary">Priimti užsakymą</button> </td>
+                <td> <button type="submit" class="btn btn-success" onclick="@Clicked">Accept the task</button> </td>
             </tr>
         }
 
     </tbody>
 </table>
 
+@if (popup)
+{
+    <AdPopUp Title="Task"
+         Text="Task accepted succesfully!" 
+         TextName="Name:"
+         TextAdress="Address:"
+         TextPhone="Phone:"
+         OnClose="@ClosePopup"
+         DialogType="AdPopUp.ModalDialogType.Ok">
+    </AdPopUp>
+}
+
 
 
 @code {
 
 
     DateTime now = DateTime.Now.ToUniversalTime();
-
+    public bool popup { get; set; } = false;
     private int n = 0;
     ReadPost json = new ReadPost();
     protected override async Task OnInitializedAsync()
     {
         json.LoadJson();
     }
-
+    private void Clicked()
+    {
+        popup = true;
+    }
+    private async Task ClosePopup(bool accepted)
+    {
+        popup = false;
+        StateHasChanged();
+        NavMan.NavigateTo("/");
```

Comment:
Maybe it should let you stay in the feed tab

Reply:
I agree